sdk/python/tests/components/test_graph_components.py

```diff
@@ -172,5 +172,16 @@ def test_handle_parsing_task_volumes_and_mounts(self):
 
 #TODO: Test task name conversion to Argo-compatible names
 
+    def test_handle_loading_graph_component_using_load_component(self):
```

Comment:
Is this needed given that graph component is not in scope?

Reply:
This is an easy test that shows that the changes worked (because graph components just cannot be converted to ContainerOp by design). Otherwise I could do type checking, but that's frown upon in python.

P.S. Graph components are in scope. See the planning doc. The item is named "Pipeline component packaging and sharing".

Reply:
Let's sync with Pavel. I won't think graph components is in the scope of the intermediate yaml. Do you mind move graph component related changes in other PRs, so it won't block your PR to convert to ContainerOp?